### PR TITLE
Avoid hardcoding the drake location

### PR DESCRIPTION
**Pull request by stonier**
I'd prefer to illustrate the practice of leaving it explicitly to the user's build configuration so they can decide where to point it at. Hardcoded like this it always ends up being a suprise when the install location is not there (and there is no agreed upon convention that insists that it should be there).

### CODE REVIEW

**Comment by jamiesnape** [comment]
> (and there
is no agreed upon convention that insists that it should be there)

Actually, the binary packages currently require that they are installed there. Make them relocatable is a work in progress.

**Comment by stonier** [comment]
Oh, that is indeed an important caveat. Nonetheless, I'd like to get away from magicking it away inside the `CMakeLists.txt`. The devs here run into all manner of problems doing so with theirs and other research projects that have magic paths and I'm trying to encourage them to handle this at build time.

**Comment by jamiesnape** [comment]
This is `CMAKE_PREFIX_PATH`, and not `CMAKE_INSTALL_PREFIX`, that we are talking about?

**Comment by jamiesnape** [comment]
`CMAKE_PREFIX_PATH` is expected to be set by projects. From the [CMake documentation](https://cmake.org/cmake/help/v3.5/variable/CMAKE_PREFIX_PATH.html):

>  By default it is empty, it is intended to be set by the project.

This is best practice.

**Comment by stonier** [comment]
@jamiesnape switched to drake_DIR

File: drake_cmake_installed/CMakeLists.txt [hunk 1]
```diff
@@ -36,7 +36,9 @@ project(drake_cmake_installed)
 include(CTest)
 include(GNUInstallDirs)
 
-list(APPEND CMAKE_PREFIX_PATH /opt/drake)
+if(drake_DIR)
+  list(APPEND CMAKE_PREFIX_PATH ${drake_DIR})
```
Comment (jamiesnape) [inline drake_cmake_installed/CMakeLists.txt]: Move to after `find_package(drake)` and remove the `if() ... endif()`.

**Comment by stonier** [comment]
Updated and added a few comments. Please check for sanity.

**Pull request closed without merging**